### PR TITLE
Include PR and issue templates

### PR DESCRIPTION
**Pull request by fernandopradocabrillo**
Include templates for new PRs and Issues

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue_bug_template.md
Comment (hdamker) [inline .github/ISSUE_TEMPLATE/issue_bug_template.md:1]: There will be no provider implementation in this repo ... so this template could be omitted.

File: .github/ISSUE_TEMPLATE/config.yml
Comment (hdamker) [inline .github/ISSUE_TEMPLATE/config.yml:1]: Do you deleted "Discussions" by intent? If you would like to have them, I'm happy to enable on the repo.

Comment (fernandopradocabrillo) [reply .github/ISSUE_TEMPLATE/config.yml:1]: No, I just copied the same ones from Sim swap and this is the first time I'm doing this so yes, please feel free to enable/add whatever you feel that should be there.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
```
Comment (hdamker) [inline .github/ISSUE_TEMPLATE/config.yml:2]:
```suggestion
contact_links:
  - name: 🗣 Subproject discussions 
    url: https://github.com/camaraproject/KnowYourCustomer/discussions
    about: Please ask and answer questions here.
```

File: .github/ISSUE_TEMPLATE/config.yml
Comment (hdamker) [reply .github/ISSUE_TEMPLATE/config.yml:1]: Ok, I have enabled discussions for the sub project and suggested the necessary change to the config below.

File: .github/ISSUE_TEMPLATE/issue_bug_template.md
Comment (hdamker) [reply .github/ISSUE_TEMPLATE/issue_bug_template.md:1]: You need just remove .github/ISSUE_TEMPLATE/issue_bug_template.md from the PR.

Comment (fernandopradocabrillo) [reply .github/ISSUE_TEMPLATE/issue_bug_template.md:1]: Done, uploaded a new commit

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (fernandopradocabrillo) [reply .github/ISSUE_TEMPLATE/config.yml:2]: Done in a new commit

**Review by hdamker — APPROVED** [review]
LGTM. @ToshiWakayama-KDDI has to approve as codeowner.

**Comment by rartych** [comment]
In order to fully support automatic issue labeling 2 labels need to be added (created): correction , subproject management.

(Label colors can be set to #DB2F1F and #515C8B respectively)

**Comment by rartych** [comment]
It looks that this PR covers issue in https://github.com/camaraproject/Governance/issues/98

**Review by ToshiWakayama-KDDI — APPROVED** [review]

**Comment by hdamker** [comment]
> In order to fully support automatic issue labeling 2 labels need to be added (created): correction , subproject management.
> 
> (Label colors can be set to #DB2F1F and #515C8B respectively)

Done.

**Pull request merged**